api/redhatopenshift/resource-manager/Microsoft.RedHatOpenshift/preview/2024-06-10-preview/openapi.json

```diff
@@ -0,0 +1,2174 @@
+          "readOnly": true
+        }
+      },
+      "required": [
+        "provisioningState",
+        "clusterVersion"
+      ]
+    },
+    "IngressProfile": {
+      "type": "object",
+      "description": "Configuration of the cluster ingress",
+      "properties": {
+        "ip": {
+          "type": "string",
+          "description": "The IP for the ingress",
+          "readOnly": true
+        },
+        "url": {
+          "type": "string",
+          "description": "The ingress url",
+          "readOnly": true
+        },
+        "visibility": {
+          "$ref": "#/definitions/Visibility",
+          "description": "The visibility of the ingress\ndetermines if the ingress is visible from the internet",
+          "x-ms-mutability": [
+            "create"
+          ]
+        }
+      },
+      "required": [
+        "ip",
+        "url",
+        "visibility"
+      ]
+    },
+    "NetworkProfile": {
+      "type": "object",
+      "description": "Network profile of the cluster",
+      "properties": {
+        "networkType": {
+          "$ref": "#/definitions/NetworkType",
+          "description": "The main controller responsible for rendering the core networking components",
+          "default": "OVNKubernetes",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "podCidr": {
+          "type": "string",
+          "description": "The CIDR of the pod IP addresses\nexample: 10.128.0.0/14",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "serviceCidr": {
+          "type": "string",
+          "description": "The CIDR block for assigned service IPs,\nexample: 172.30.0.0/16",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "machineCidr": {
+          "type": "string",
+          "description": "from which to assign machine IP addresses,\nexample: 10.0.0.0/16",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "hostPrefix": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Network host prefix which is defaulted to 23 if not specified.",
+          "default": 23,
+          "x-ms-mutability": [
+            "create"
+          ]
+        }
+      },
+      "required": [
+        "podCidr",
+        "serviceCidr",
+        "machineCidr"
+      ]
+    },
+    "NetworkType": {
+      "type": "string",
+      "description": "The cluster network type",
+      "enum": [
+        "OpenShiftSDN",
+        "OVNKubernetes",
+        "Other"
+      ],
+      "x-ms-enum": {
+        "name": "NetworkType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "OpenShiftSDN",
+            "value": "OpenShiftSDN",
+            "description": "The SDN network plugin for the OpenShift cluster\n- https://docs.openshift.com/container-platform/4.12/networking/openshift_sdn/about-openshift-sdn.html"
+          },
```

Comment:
```suggestion
```
My understanding is that OpenShiftSDN will not be an option for ARO HCP

Reply:
Yes, it is removed